### PR TITLE
EXSWHTEC-1 - Implement additional negative parameter tests for hipIpcGetEventHandle and hipIpcOpenEventHandle

### PR DESCRIPTION
**Pull request by mirza-halilcevic**

### CODE REVIEW

File: tests/catch/multiproc/hipIpcEventHandle.cc [hunk 1]
```diff
@@ -353,6 +357,22 @@ TEST_CASE("Unit_hipIpcEventHandle_ParameterValidation") {
       REQUIRE(false);
     }
   }
+
+  SECTION("Open handle in process that created it") {
+    hipIpcEventHandle_t event_handle;
+    hipEvent_t event1, event2;
+    HIP_CHECK(hipEventCreateWithFlags(&event1, hipEventDisableTiming | hipEventInterprocess));
+    HIP_CHECK(hipIpcGetEventHandle(&event_handle, event1));
+    HIP_CHECK_ERROR(hipIpcOpenEventHandle(&event2, event_handle), hipErrorInvalidContext);
```
Comment (rorake) [inline tests/catch/multiproc/hipIpcEventHandle.cc:367]: Destroy created event1 using hipEventDestroy

File: tests/catch/multiproc/hipIpcEventHandle.cc [hunk 2]
```diff
@@ -353,6 +357,22 @@ TEST_CASE("Unit_hipIpcEventHandle_ParameterValidation") {
...
+  }
+
+  SECTION("Event created with no flags") {
+    hipEvent_t event;
+    hipIpcEventHandle_t event_handle;
+
+    HIP_CHECK(hipEventCreate(&event));
+    HIP_CHECK_ERROR(hipIpcGetEventHandle(&event_handle, event), hipErrorInvalidResourceHandle);
+  }
```
Comment (rorake) [inline tests/catch/multiproc/hipIpcEventHandle.cc:380]: Destroy created event using hipEventDestroy

File: tests/catch/multiproc/hipIpcEventHandle.cc (hunk 1 above)
Comment (mirza-halilcevic) [reply tests/catch/multiproc/hipIpcEventHandle.cc:367]: The suggestion is implemented. Please proceed with the review.

File: tests/catch/multiproc/hipIpcEventHandle.cc (hunk 2 above)
Comment (mirza-halilcevic) [reply tests/catch/multiproc/hipIpcEventHandle.cc:380]: The suggestion is implemented. Please proceed with the review.

**Review by rorake — DISMISSED** [review]

**Comment by mangupta** [comment]
@music-dino / @mirza-halilcevic : Some tests fail on CI. Details shared separately.

**Comment by music-dino** [comment]
> @music-dino / @mirza-halilcevic : Some tests fail on CI. Details shared separately.

PR updated, please proceed with merge.

**Pull request merged**